README.md

```diff
@@ -493,16 +495,83 @@ If it is not possible to use existing nf-core modules, you can create your own m
+
+```
+docker.registry =  'quay.io'
+```
+
 For more information, see the [Nextflow containers][] documentation and the [nf-core modules software requirements][] guide.
 
+### 5.2.2 Configuring Module Software with Private or Alternate Container Registries
+
+To configure a private or alternate container registry, the standard practice is to set the `.registry` option to point to the desired registry in the `nextflow.config` file. For example:
+
+```
+docker.registry = 'container-registry.com'
+```
+
+This setting automatically prefixes each container name with the specified registry URL. In the above example, `biocontainers/fastqc:0.11.9--0` is transformed into `container-registry.com/biocontainers/fastqc:0.11.9--0`, ensuring that the container is downloaded from `container-registry.com`.
+
+In IRIDA Next Nextflow pipelines, we have chosen to adhere to nf-core standards by designating `quay.io` as the default container registry. This setup ensures that containers, such as `biocontainers/fastqc:0.11.9--0`, are automatically referenced as `quay.io/biocontainers/fastqc:0.11.9--0`. While this configuration simplifies container management, it can be restrictive when the pipeline requires containers from multiple registries, such as both `quay.io` and `docker.io`, since only one default registry can be configured at a time. Introducing flexibility in sourcing containers from different registries becomes valuable in situations where specific containers are only available in certain registries, or where there are preferences or restrictions on registry use.
+
+To address this, we use `process.ext.override_configured_container_registry` to manage the selection of container registries dynamically by adding the following line to the `nextflow.config` file:
+
+**nextflow.config**
+
+```
+// Override the default Docker registry when required
+process.ext.override_configured_container_registry = true
+```
+
+This custom extension allows us to define configurations beyond the standard Nextflow syntax, tailored to the needs of specific processes in the workflow. When a process runs, the value of `process.ext.override_configured_container_registry` is accessed within the `container` directive as `task.ext.override_configured_container_registry`. The `task` context refers to the specific instance of a process execution, encompassing all relevant attributes and extensions for that instance. This setup ensures that any custom settings, such as overriding the default container registries, are correctly applied during the execution of the pipeline.
+
+To specify an alternative registry for a process, include a conditional check with `task.ext.override_configured_container_registry` in the nested ternary operator of the `container` directive. For example:
+
+```
```

Comment:
This looks really good! I will go implement this shortly! 

Is it possible to label this example in the table of contents as a `quickstart` or `example` section so that readers know immediately which section to jump too?

Reply:
For sure, love this idea!
Done here: [616f572](https://github.com/phac-nml/pipeline-standards/pull/24/commits/616f5723878b75b9cbcec9f09d38d7b613336283)